packages/lutris-deb/lutris-deb.pacscript

```diff
@@ -7,3 +7,8 @@ depends=("python3-gi-cairo")
 pkgdesc="open source gaming platform"
 hash="75267ff9ab1e21b5ca8ca428247791460b6957c7d758a2803fe9c56681390415"
 maintainer="Oren Klopfer <oren@taumoda.com>"
+
+pre_install() {
```

Comment:
Use `prepare()` for this, as `pre_install()` is run after `package()`, per https://github.com/pacstall/pacstall/wiki/Pacscript-101#pre_install